pytri/__init__.py

```diff
@@ -282,6 +289,33 @@ def graph(self, data, r=0.15, c=0xbabe00, name=None) -> str:
             "colors": c
         }, name=name)
 
+
+    def large_graph(self, data, r=2, c=0x000000, name=None) -> str:
+        """
+        Add a large graph to the visualizer using the GPU particle system.
+
+        Arguments:
+            data (networkx.graph)
+
+        Returns:
+            name of layer (string)
+        """
+        if isinstance(data, nx.Graph):
+            data = json_graph.node_link_data(data)
+        x_vals = [n['x'] for n in data['nodes']]
+        y_vals = [n['y'] for n in data['nodes']]
+        z_vals = [n['z'] for n in data['nodes']]
+        minmax = [max(x_vals), min(x_vals), max(y_vals), min(y_vals), max(z_vals), min(z_vals)]
```

Comment:
You call this `minmax` but the order is `maxmin`. Convention is (`min`, `max`), so you're half right!

I'll prob fix this in a future PR, but wanted to leave this here for awareness.